### ISSUE TITLE
Documentation site link pointing to github.com instead of github.io

### ISSUE THREAD

**Issue by paulj** [kschiess/parslet#215]
We've recently started using Parselet, and have it found it to be great! The documentation is really good too, but the Github link to it is a little broken since Github moved the Github pages content to the `.io` domain instead of under `.com`. I believe this is just repository configuration, otherwise I'd raise a pull request for it 😄

**Comment by kschiess** [comment]
Please, PR would be welcome.

**Comment by paulj** [comment]
Hi @kschiess - I've checked, and this is just a repository configuration option.

On https://github.com/kschiess/parslet, in the top right corner with the 'About' panel, you should have a little gear icon in this panel:

<img width="323" alt="image" src="https://user-images.githubusercontent.com/52834/206036959-b2d0b157-cf45-4d0b-b012-68acc18e62ce.png">

Clicking that shows the 'Website' box that would let you change the link https://kschiess.github.io/parslet/ which looks to be the right one now :)

**Comment by dscho** [comment]
@kschiess indeed, as initially stated this cannot be done via a PR. You will have to direct your own browser to https://github.com/kschiess/parslet, then on the right side click on this button (which only you can see, so it's really up to you to edit this):

![Image](https://github.com/user-attachments/assets/a7c5a50f-6f02-44ea-93d5-8f0efbe00d24)

And then, in the dialog that pops up, you will have to edit this part so it says "io" instead of "com":

![Image](https://github.com/user-attachments/assets/d8bb1192-0a4b-4aa6-9673-4030f673556e)

Unfortunately only you can do this. The `.com` link is broken, so it will need to be done.
